[2024-07-03T21:21:13.937+0000] {scheduler_job_runner.py:769} ERROR - Executor reports task instance <TaskInstance: exchange_rate_dag.run_script scheduled__2024-01-12T00:00:00+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
[2024-07-03T22:29:49.644+0000] {scheduler_job_runner.py:769} ERROR - Executor reports task instance <TaskInstance: exchange_rate_dag.run_script scheduled__2024-01-12T00:00:00+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
[2024-07-04T01:04:17.911+0000] {scheduler_job_runner.py:769} ERROR - Executor reports task instance <TaskInstance: exchange_rate_dag.run_script scheduled__2024-01-12T00:00:00+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
